mod/01_rating/set/type/metric_value.rb

```diff
@@ -190,36 +223,62 @@ def legend args
     subformat(card.metric_card)._render_legend args
   end
 
+  def currency
+    return unless (value_type = Card["#{card.metric_card.name}+value type"])
```

Comment:
again, if this is in +value_type, we can skip rechecking that we have a value type card.